server/src/concepts/height/useCases/addNewHeight.js

```diff
@@ -0,0 +1,16 @@
+import createHeight from "../repositories/commands.js";
+
+const addNewHeight = async (data) => {
+  const addHeight = await createHeight({
```

Comment:
You do not have to save it here. Just trigger this function with our data object and return its result. Saving would be triggered inside commands
```suggestion
  return await createHeight(data)
```

You do not need this try below etc.